app/models/miq_database.rb

```diff
@@ -1,57 +1,14 @@
-require 'util/postgres_admin'
-
 class MiqDatabase < ApplicationRecord
-  REGISTRATION_DEFAULT_VALUES = {
-    :registration_type   => "sm_hosted",
-    :registration_server => "subscription.rhn.redhat.com"
-  }.freeze
-
-  include AuthenticationMixin
```

Comment:
This makes me think we need a data migration to drop any authentications that are in the database and attached to the MiqDatabase record.

Reply:
See https://github.com/ManageIQ/manageiq-schema/pull/588